DataServer/server.py

```diff
@@ -183,16 +193,23 @@ async def runRTPServer(self, port: int):
         file_path = os.path.abspath(os.path.join(data_dir, device_path, fname))
         file_dir = os.path.dirname(file_path)
         pathlib.Path(file_dir).mkdir(parents=True, exist_ok=True)
-        
+
+        ff_stats_path = pathlib.Path(self.ff_log_dir, device_path, "stats.log")
+        ff_info_path = pathlib.Path(self.ff_log_dir, device_path, "info.log")
+        script_path = "-m ASM_utils.ffmpeg.split_log"
+
         cmd = (f'ffmpeg -i tcp://@:{port}?listen -c copy -flags +global_header'
                f' -f segment -segment_time {self._config.video_increment_s} -strftime 1 '
-               f'-reset_timestamps 1 {file_path}')
+               f'-reset_timestamps 1 {file_path} '
+               f' 2>&1 | {sys.executable} {script_path} {ff_stats_path} {ff_info_path}'
+            )
         proc_out = asyncio.subprocess.PIPE
         proc_err = asyncio.subprocess.PIPE
         self._log.info(f'Started ffmpeg with command: {cmd}')
         proc = await asyncio.create_subprocess_shell(cmd, stdout=proc_out,
                                                      stderr=proc_err)
         self._log.info(f'RTP Server on port {port} started outputting to {file_dir}')
+        self._log.info(f"FFmpeg logging to: {os.path.join(self.ff_log_dir, device_path)}")
```

Comment:
Why not simply use `ff_stats_path.parent`?

Reply:
ff_log_dir already exists and is used to create ff_stats_path